reference/arcjet.html.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Application Security by Arcjet
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+[Arcjet](https://arcjet.com) is a security layer that allows developers to protect their apps with just a few lines of code. Implement rate limiting, bot protection, email validation, and defense against common attacks.
+
+Arcjet is installed as a dependency in your application and doesn't require an agent or any additional infrastructure. The SDK is currently available for JS applications, with support for other languages coming soon.
+
+## Get started
+
+To configure your application to use Arcjet, run this command from your Fly.io application directory:
+
+```cmd
+flyctl ext arcjet create
+```
+
+Then set the `ARCJET_KEY` secret on your app.
```

Comment:
If you run the command in your app directory, the key will be set automatically. If you set it outside the app directory, you'd need to manually run this command.